src/Common/Constants.ts

```diff
@@ -353,7 +355,7 @@ export const StyleConstants = require("less-vars-loader!../../less/Common/Consta
 
 export class Notebook {
   public static readonly defaultBasePath = "./notebooks";
-  public static readonly heartbeatDelayMs = 5000;
+  public static readonly heartbeatDelayMs = 20000;
```

Comment:
>20000 [](http://example.com/codeflow?start=44&length=5)

20 ms ? what all is this used for? is this again too aggresive? may be we can do once per minute?

Reply:
Updated to Reconnect